api/client/run.go

```diff
@@ -121,8 +121,8 @@ func (cli *DockerCli) initSpecialVolumes(config *container.Config, hostConfig *c
 	var (
 		initConfig     *container.Config
 		initHostConfig *container.HostConfig
-		execJobs       []string
-		execID         string
+		errCh          chan error
+		execCount      uint32
```

Comment:
It is more about waiting in parallel rather than issuing requests in parallel. I'll fix up the changelog.